drivers/volume/common.go

```diff
@@ -658,6 +658,14 @@ func (d *DefaultDriver) GetAutoFsTrimStatus(pxEndpoint string) (map[string]api.F
 	}
 }
 
+// GetAutoFsTrimUsage get autofstrim  usage stats
+func (d *DefaultDriver) GetAutoFsTrimUsage(pxEndpoint string) (map[string]*api.FstrimVolumeUsageInfo, error) {
```

Comment:
Why do we need `pxEndpoint` string when it is not even being used in the function?

Reply:
it is a interface method, so signature should be same for all implemented functions